### DISCUSSION TITLE
discussion

### DISCUSSION THREAD

**Discussion by jvcByte** [drknzz/GitHub-Achievements#373] [General]
how do get a galaxy brain?

**Comment by PlantBasedStudio** [comment]
Accept my answer and see

**Reply by jvcByte** [reply]
can't see the mark as answer btn

**Comment by ghost** [comment]
In order to get galaxy brain you need your responses to be accepted as answers twice. Discussion authors and users with the triage role or greater for a repository can mark a comment as the answer to a discussion in the repository.

**Comment by smarenkov** [comment]
The answered mark is missing here

**Comment by mohitprajapat2001** [comment]
Python is shortcut to programming change my mind 🤯

**Comment by zuberkhan01st** [comment]
Make sure to select my answer and then close the discussion, then it will reflect on your profile
